### PR TITLE
Fix: elevator and door instances don't reference Revolution

### PR DESCRIPTION
**Pull request by craftablescience**
What it says in the title. Elevators should work now:tm:, and doors aren't broken anymore. Loads of invalid instance names are fixed.

Oh, and I removed the portal spawners with missing models, I assume they referred to a Revolution model.

### CODE REVIEW

**Review by Luke18033 — CHANGES_REQUESTED** [review]
Mostly looks good, except the elevator soundscape instances have a bunch of disabled visgroups that seem to be from an early Valve map. Those should be removed.

File: soundscapes/elevator_dirty_departure_soundscape.vmf [hunk 1]
```diff
@@ -0,0 +1,7324 @@
+versioninfo
+{
+	"editorversion" "400"
+	"editorbuild" "5157"
+	"mapversion" "858"
+	"formatversion" "100"
+	"prefab" "0"
+}
+visgroups
```
Comment (Luke18033) [inline soundscapes/elevator_dirty_departure_soundscape.vmf:9]: The disabled visgroups are also here.

**Review by Luke18033 — CHANGES_REQUESTED** [review]
There are some disabled visgroups leftover in the elevator soundscape instances, which should probably be removed.

**Comment by craftablescience** [comment]
Leave it to Valve to hide what looks like half of a testing level inside two soundscape instances

**Review by CitadelCore — APPROVED** [review]

**Review by Luke18033 — APPROVED** [review]

**Pull request merged**